draft-irtf-cfrg-hpke.md

```diff
@@ -1059,7 +1134,7 @@ secrets of senders and recipients. The desired security goals are
 detailed below:
 
 * Message secrecy: Confidentiality of the sender's messages against
-  chosen ciphertext attacks, i.e., IND-CCA2 security
```

Comment:
I removed this, because IND-CCA2 is not well-defined for the authenticated modes. ABHKLR20 uses different names (Outsider-CCA and Insider-CCA), so we could add them here, but there is no security notions for the PSK yet.